### DISCUSSION TITLE
Drehzahlschwankungen bei Stufenlosgetrieben

### DISCUSSION THREAD

**Discussion by Bedda1989** [Mogli12/KeyboardSteer#831] [Q&A]
Wunderschöner Guten Tag sei gewünscht!

Gleich vorweg, ich habe schon sowohl im GIANTS Forum, als auch auf deren "Bug - Repport - Seite" und auch in deren DC direkt probiert das "Problem" anzusprechen bzw. zu reporten, leider ohne Erfolg. 
Da ich ein riesen Fan von VCA bin und auch schon im vorherigen Teil war, hab ich mich gefragt, vielleicht kann mir hier weitergeholfen , bzw. das Problem durch das VCA Skript behoben werden.
Bestimmt ist schon vielen aufgefallen, dass wenn man Feldarbeiten mit einem  stufenlos Traktor (egal welchen) macht, das es hier in regelmäßigen Abständen zu teilweise gravierenden Drehzahlschwankungen, als auch zu Geschwindigkeitsschwankungen kommt. Vor allem wenn man den Tempomat nutzt. Wenn man, so wie ich Bezug zur realen Landwirtschaft hat, weiß man, dass diese extremen Schwankungen unrealistisch sind, vor allem bei zapfwellengetrieben Tätigkeiten, wo eine fixe konstante Drehzahl erforderlich ist. 
Seit ich VCA verwende, finde ich, hat sich das Problem gebessert, vor allem, wenn man die Drehzahl selbst vorgibt (Bei Schaltern ist dieses Problem eher zu vernachlässigen finde ich) Allerdings sind immer noch gelegentlich ordentliche Drops dabei, wo für kurze Zeit die Drehzahl von z.B. 1800 Umdrehung auf 1500 runter droppt. 
Und ich wollte jetzt fragen, ob das irgendwie behoben werden kann, oder von mir aus sogar so zurückgestellt werden kann, wie es  damals beim Standard "Stufenlosgetriebe" aus dem LS19 (Fixe Drehzahl bei Tempomat und eingeschalteter Zapfwelle) war ... Vielleicht kann mir hier jemand weiterhelfen oder weiß, wie das Problem behoben werden könnte. 
Mit freundlichen Grüßen
Peter

**Comment by Mogli12** [comment]
Hallo Peter,
ich glaube, dass es hier 2 mögliche Erklärungen für die Drehzahlschwankungen gibt.

1. FS22 packt auf die Drehzahl noch absichtlich Schwankungen oben drauf,
2. Die Motorlast schwankt sehr stark. In FS22 überschreibt VCA das Getriebe nicht, sondern gibt nur engere Drehzahlbereiche vor. Ich habe in FS19 sehr viel Aufwand getrieben um die Motorlast optimal zu glätten.

Viele Grüße,
Stefan

**Reply by Bedda1989** [reply]
Hallo Stefan,

stellt sich halt die Frage ob sich das irgendwie fixen lässt, oder ob Giants selbst mal drauf kommt, dass da was nicht stimmt. Bzw. würde es was bringen, wenn man die Drehzahlbereiche noch enger stellt, ich muss ja gestehen, bei Scripten bin ich ja leider ein kompletter Anfänger, also sorry, falls diese Frage irgendwie Schwachsinn war.

Freundliche Grüße

Peter
